maps/sccv_horizon/code/sccv_horizon.dm

```diff
@@ -132,3 +132,59 @@
 	allow_borgs_to_leave = TRUE
 
 	warehouse_basearea = /area/operations/storage
+
+/datum/map/sccv_horizon/send_welcome()
+	var/obj/effect/overmap/visitable/ship/horizon = SSshuttle.ship_by_type(/obj/effect/overmap/visitable/ship/sccv_horizon)
+
+	var/welcome_text = "<center><img src = scclogo.png><br />[FONT_LARGE("<b>SCCV Horizon</b> Ultra-Range Sensor Readings:")]<br>"
+	welcome_text += "Report generated on [worlddate2text()] at [worldtime2text()]</center><br /><br />"
+	welcome_text += "<hr>Current sector:<br /><b>[SSatlas.current_sector.name]</b><br /><br>"
+
+	if (horizon) //If the overmap is disabled, it's possible for there to be no Horizon.
+		var/list/space_things = list()
+		welcome_text += "Current Coordinates:<br /><b>[horizon.x]:[horizon.y]</b><br /><br>"
+		welcome_text += "Next system targeted for jump:<br /><b>[SSatlas.current_sector.generate_system_name()]</b><br /><br>"
+		var/last_visit
+		var/current_day = time2text(world.realtime, "Day")
+		switch(current_day)
+			if("Monday")
+				last_visit = "1 day ago"
+			if("Tuesday")
+				last_visit = "2 days ago"
+			if("Wednesday")
+				last_visit = "3 days ago"
+			if("Thursday")
+				last_visit = "4 days ago"
+			if("Friday")
+				last_visit = "5 days ago"
+			if("Saturday")
+				last_visit = "6 days ago"
+			if("Sunday")
+				last_visit = "1 week ago"
+		welcome_text += "Last port visit: <br><b>[last_visit]</b><br><br>"
```

Comment:
Wonder if this would be wanted, worded in this way, considering there might be times in the future where the horizion is away from a port for a longer period of time?

Reply:
Right now, yes